src/index.tsx

```diff
@@ -13,6 +13,12 @@ const LazyConfig = React.lazy(async () => import('./pages/config.jsx'))
 const LazyHelperUi = React.lazy(async () => import('./pages/helper-ui.jsx'))
 const LazyRedirectPage = React.lazy(async () => import('./pages/redirect-page.jsx'))
 const LazyInterstitial = React.lazy(async () => import('./pages/redirects-interstitial.jsx'))
+const LazyServiceWorkerErrorPage = React.lazy(async () => import('./pages/errors/no-service-worker.jsx'))
+
+let ErrorPage: null | React.LazyExoticComponent<() => React.JSX.Element> = null
```

Comment:
@SgtPooki I get lint error on this line

Reply:
yea i had to fix some internet/cli issues here but i've got the branch pulled down now and am working on fixing things